great_expectations/util.py

```diff
@@ -1316,6 +1317,34 @@ def is_nan(value: Any) -> bool:
         return True
 
 
+def isclose(
+    operand_a: Union[datetime.datetime, Number],
+    operand_b: Union[datetime.datetime, Number],
+    rtol=1.0e-5,
+    atol=1.0e-8,
+    equal_nan=False,
+) -> bool:
+    """
+    Checks whether or not two numbers (or timestamps) are approximately close to one another.
```

Comment:
Would it be possible to document the args here? `rtol` and `atol` confuse me, even after reading the `numpy` documentation.